Builds/VisualStudio/stellar-core.vcxproj

```diff
@@ -1112,6 +1116,27 @@ exit /b 0
       <OutputItemType Condition="'$(Configuration)|$(Platform)'=='DebugNoPostgres|x64'">ClInclude</OutputItemType>
       <OutputItemType Condition="'$(Configuration)|$(Platform)'=='Release|x64'">ClInclude</OutputItemType>
     </CustomBuild>
+    <CustomBuild Include="..\..\src\xdr\Stellar-internal.x">
```

Comment:
this still references `src\xdr` instead of curr/next

Reply:
(this causes the 2 files that include `stellar-internal.h` to be recompiled all the time as msbuild cannot figure out dependencies)

Reply:
Thanks for the remainder. I'm aware of this, just wanted to postpone the update to rebase.

Reply:
Rebased and updated this.